src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -106,11 +108,16 @@ private OpenLoadFlowParameters getParametersExt(LoadFlowParameters parameters) {
         return parametersExt;
     }
 
+    private static SlackBusSelector getSlackBusSelector(Network network, LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt) {
+        return parameters.isReadSlackBus() ? new NetworkSlackBusSelector(network, parametersExt.getSlackBusSelector())
+                                           : parametersExt.getSlackBusSelector();
```

Comment:
I don't think the `isReadSlackBus` should be priority... I would say:
- if parametersExt.getSlackBusSelector is set, then use it
- if parameters.isReadSlackBus is set, then use NetworkSlackBusSelector
- default: use the default strategy